database/migrations/2021_03_14_093812_add_read_column_tickets_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddReadColumnTicketsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('tickets', function (Blueprint $table) {
+            $table->tinyInteger('user_read')->nullable()->after('staff_id');
```

Comment:
Would it not be more appropriate if these two new columns where a boolean with a default of 0 or false which would also allow you to get rid of these lines. https://github.com/HDInnovations/UNIT3D-Community-Edition/blob/149f3a3c09dcac2cfbc2a0cc9f436e51ebca24b4/app/Http/Controllers/CommentController.php#L556-L561 ?

Reply:
@HDVinnie I have used the table "private_messages" as template, where the "read" column is a tinyint. besides having less code, i dont see much of a performance improvement in using a bool with default 0.
However, I'm not that experienced yet. Decision's up to you.